### PR TITLE
feature/AIRENEIS-43: Created the structs of the model

### PR DESCRIPTION
**Pull request by BarnabePILLIAUDIN**
# WIP

# Done 

- Added the structs of the model
- Replaced Price by Price with taxes and Price without taxes to help us when we will do invoices
- Added a field isActive to address for history purpose so the previous orders are not affected when the address changes
- Updated doc

# CLOSED WILL BE MERGED IN ANOTHER PR

### CODE REVIEW

File: back/main.go [hunk 1]
```diff
@@ -23,12 +22,16 @@ func main() {
...
 	})
 
-	secret, err := config.Handler.VaultClient.ReadSecret("appname_secret_version")
+	db, err := neon.InitDB(config.Handler.VaultClient)
 	if err != nil {
-		println(err)
+		log.Fatal("Unable to connect to DB", err.Error())
+	}
+
+	if err := config.DB.AutoMigrate( /* PLACE HERE THE STRUCTS OF THE MODEL*/ ); err != nil {
```
Comment (avetisk) [inline back/main.go:30]: Placeholder comments are for L.

image

File: back/main.go [hunk 2]
```diff
@@ -23,12 +22,16 @@ func main() {
...
+		log.Fatal("Unable to run automigrate", err.Error())
 	}
 
-	fmt.Println(secret)
+	log.Info(neon.CheckVersion(db))
```
Comment (avetisk) [inline back/main.go:34]: What's the purpose of this log?

Comment (barnabepilliaudin) [reply back/main.go:34]: It's to check that we can connect to the Neon db

Comment (avetisk) [reply back/main.go:34]: Sure, but then it's like a `console.log`: either use persistent log (e.g log server) or use a conditional logger (e.g that would print only if DEBUG=1).

Comment (cedricgautier) [reply back/main.go:34]: For info, this was removed. Thanks

**Pull request closed without merging**